src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldTransformers/ColorFieldTransformer.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import React from 'react';
+import log from 'loglevel';
+import type {Node} from 'react';
+import type {FieldTransformer} from '../types';
+import colorFieldTransformerStyles from './colorFieldTransformer.scss';
+
+export default class ColorFieldTransformer implements FieldTransformer {
+    transform(value: *): Node {
```

Comment:
i think we dont want to log an error if the value is not set 🙂 

```suggestion
    transform(value: *): Node {
        if (!value) {
            return null;
        }
```